articles/native-platforms/cordova/01-login.md

```diff
@@ -14,10 +14,12 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-cordova',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-cordova-samples',
+  githubUrl: 'https://github.com/auth0-samples/auth0-cordova-samples/tree/master/00-Starter-Seed',
```

Comment:
@mehreencs87 Can you please set `githubUrl` to `https://github.com/auth0-samples/auth0-cordova-samples/tree/master/00-Starter-Seed/basic-sample`? Thanks!